### PR TITLE
examples: add a yaml for smbd+winbind pod

### PR DESCRIPTION
**Pull request by phlogistonjohn**
This adds a basic, but functional, example of running a pod in kubernetes with both sbmd and winbind for use as a domain member smb server.

The YAML contains a ConfigMap, secret, and pod configuration. One must edit parts of the yaml (and the embedded JSON) to match the  user's domain.

This currently needs a new build of the samba server container to work thus it won't work without the commit bumping the version of sambacc in use. If you want I can split these into separate PRs. I wasn't sure if that was needed as this is "just" an example.

### CODE REVIEW

File: examples/kubernetes/sambadmpod.yml [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+# You can tweak the samba-sharedir volume under "volumes" at the end
+# of the pod yaml below. Use this to toggle between a temporary share
+# or a PVC backed share.
+#
+# NOTE: All settings below can be modified but those mentioned above
+# will get you a running container more quickly, feel free to start
+# there and then begin tinkering!
+#
+---
+# Configuration for thae samba domain member pod.
```
Comment (obnoxxx) [inline examples/kubernetes/sambadmpod.yml:26]:
```suggestion
# Configuration for the samba domain member pod.
```

**Review by obnoxxx — CHANGES_REQUESTED** [review]
small typo

File: examples/kubernetes/sambadmpod.yml (hunk 1 above)
Comment (phlogistonjohn) [reply examples/kubernetes/sambadmpod.yml]: Fixed.

**Review by obnoxxx — APPROVED** [review]
lgtm

**Pull request merged**